src/main/resources/templates/casInstitutionLoginView.html

```diff
@@ -30,11 +30,8 @@
                         </table>
                     </section>
 
-                    <section class="text-without-mdi text-center text-bold text-large margin-large-vertical">
-                        <span th:utext="#{screen.institutionlogin.tips}"></span>
-                    </section>
-
                     <section class="card-message">
+                        <h2 th:utext="#{screen.institutionlogin.heading}"></h2>
                         <p th:unless=${osfCasLoginContext.institutionId} th:utext="#{screen.institutionlogin.message.select}"></p>
                         <p th:if=${osfCasLoginContext.institutionId} th:utext="#{screen.institutionlogin.message.auto}"></p>
                     </section>
```

Comment:
The new FE work made the left-aligned "heading" into center-aligned "tips" under the banner. After comparison, I feel your left-aligned "heading" approach looks much better since institution pages have many left-aligned elements. Thus, I applied yours to the default institution page as well.